src/main/java/frc/robot/subsystems/Vision.java

```diff
@@ -0,0 +1,86 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.Relay;
+import edu.wpi.first.wpilibj.Relay.Value;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Vision extends SubsystemBase {
+    public enum VisionMode {
+        INTAKE,
+        SHOOTER,
+        HUBFINDER,
+    }
+
+    private static final double[] EMPTY_TARGET_INFO = new double[] {0.0,0.0,0.0,0.0,0.0,0.0,0.0};
+
+
+    private Relay m_spike;
+    private DriveTrain m_driveTrain;
```

Comment:
Why does the Vision system need the Drivetrain?

Reply:
For simulation

Reply:
Then you should add that as a comment.